src/server/replica.cc

```diff
@@ -428,7 +428,7 @@ error_code Replica::InitiateDflySync() {
   auto partition = Partition(num_df_flows_);
   shard_set->pool()->AwaitFiberOnAll([&](unsigned index, auto*) {
     for (auto id : partition[index]) {
-      if (ec = shard_flows_[id]->StartFullSyncFlow(&sb))
+      if ((ec = shard_flows_[id]->StartFullSyncFlow(&sb)))
```

Comment:
maybe the new still "if" with "if (ec = shard_flows_[id]->StartFullSyncFlow(&sb); ec) {..} would be more readable? right now this look a little bit like a classic C bug where you're assign and then git the side effect from assignment.

Reply:
I fully agree, but do we support c++20?